app.js

```diff
@@ -1,98 +1,193 @@
+  console.log('Welcome to the Real World.');
 }
 
 // use yesterday to help
+var correctAnswer = 0;
 
 var name = prompt('What is your name?');
 console.log('The user\'s name is ' + name);
 
 //Question One
-var answerOne = prompt('1. ' + name + ', this is a yes or no question, so please answer with a Y or N. Is Abigail\'s middle name is Margaret?').toUpperCase();
-if(answerOne === 'N') {
+var answerOne = prompt('Question 1. ' + name + ', this is a yes or no question, so please answer with a Y or N. Is Abigail\'s middle name is Margaret?').toUpperCase();
+
+function answerOneFunc(answerOne){
+  if(answerOne === 'N') {
+    return true;
+  } else {
+    return false;
+  }
+}
+if (answerOneFunc(answerOne)) {
   console.log('Correct. For Question One, the user answered ' + answerOne + '.');
+  alert('Good guess! Her middle name is Lauren, like Lauren Conrad, from The Hills, which she loves.');
+  correctAnswer ++;
 } else {
   console.log('Incorrect. For Question One, the user answered ' + answerOne + '. The answer is N.');
+  alert('Wrong but on the right track. Like her first name and her last name, Abigail\'s middle name is painfully anglosaxon. ' + 'Her middle name is Lauren.');
 }
 
-if (answerOne === 'N') {
-  alert('Good guess! Her middle name is Lauren, like Lauren Conrad, from The Hills, which she loves.');
-} else {
-  alert('Wrong but on the right track. Like her first name and last name, Abigail\'s middle name is painfully unidentifying and white. ' + 'Her name is Lauren.');
+//Question Two
+var answerTwo = prompt('Question 2. ' + name + ', here\'s another yes or no question. Did Abigail live in the UK last year?').toUpperCase();
+
+function answerTwoFunc(answerTwo){
+  if(answerTwo === 'Y') {
+    return true;
+  } else {
+    return false;
+  }
 }
 
-//Question Two
-var answerTwo = prompt('2. ' + name + ', here\'s another yes or no question. Did Abigail live in the UK last year?').toUpperCase();
-if(answerTwo === 'Y') {
+if(answerTwoFunc(answerTwo)){
   console.log('Correct. For Question Two, the user answered ' + answerTwo + '.');
+  alert('True! The trauma of 20 years of Boston winters finally took its toll and she peaced out. She lived in London    studying at LSE for 10 months while completing her junior year.');
+  correctAnswer ++;
 } else {
   console.log('Incorrect. For Question Two, the user answered ' + answerTwo + '. The answer is Y.');
+  alert('Okay. I mean, you\'re wrong. But okay.');
 }
 
-if (answerTwo === 'Y') {
-  alert('True! The trauma of 20 years of Boston winters finally took its toll and so she fled back to the motherland. She lived in London studying at LSE for 10 months while completing her junior year.');
-} else {
-  alert('Okay. I mean, you\'re wrong. But I still respect you.');
+//Question Three
+var answerThree = prompt('Question 3. You know the drill, ' + name + '. Y or N. Does Abigail enjoy biking?').toUpperCase();
+
+function answerThreeFunc(answerThree){
+  if(answerOne === 'N') {
```

Comment:
looks like a mistake we didn't catch! The answer to #1 was also 'N' so it didn't come up. Will make change